modules/sd_schedulers.py

```diff
@@ -117,12 +117,15 @@ def ddim_scheduler(n, sigma_min, sigma_max, inner_model, device):
 
 
 def beta_scheduler(n, sigma_min, sigma_max, inner_model, device):
-    # From "Beta Sampling is All You Need" [arXiv:2407.12173] (Lee et. al, 2024) """
+    # From "Beta Sampling is All You Need" [arXiv:2407.12173] (Lee et. al, 2024)
     alpha = shared.opts.beta_dist_alpha
     beta = shared.opts.beta_dist_beta
-    timesteps = 1 - np.linspace(0, 1, n)
-    timesteps = [stats.beta.ppf(x, alpha, beta) for x in timesteps]
-    sigmas = [sigma_min + (x * (sigma_max-sigma_min)) for x in timesteps]
+    curve = [stats.beta.ppf(x, alpha, beta) for x in np.linspace(1, 0, n)]
```

Comment:
I think this needs the endpoint removed, unless [this](https://github.com/comfyanonymous/ComfyUI/blob/d9a87c1e6a390eb0e915b544e35baf4d807919db/comfy/samplers.py#L448) is also wrong.
```suggestion
    curve = [stats.beta.ppf(x, alpha, beta) for x in np.linspace(1, 0, n, endpoint=False)]
```